druid/src/scroll_component.rs

```diff
@@ -475,24 +444,4 @@ impl ScrollComponent {
             self.reset_scrollbar_fade(|d| ctx.request_timer(d), &env);
         }
     }
-
-    /// Helper function to paint a closure at the correct offset with clipping and scrollbars
-    pub fn paint_content(
```

Comment:
This function has value for scroll-able widgets which are not containers for children so I'm inclined to keep it even though Scroll no longer uses it

Reply:
That seems to be a different standard for inclusion than everything else ( might be useful for some people but not used here)? I guess if there was an example that used it... (not that I want to impose work on anyone)

Reply:
Yeah honestly I'm not *very* strongly opposed to removing this as it's honestly not much code

Reply:
:shrug: no strong opinion from me, but since I already removed it I'm going to be lazy and leave it removed for now. Can always put it back in later.

Reply:
Fair enough!